core/src/main/java/io/confluent/kafka/schemaregistry/id/IncrementalIdGenerator.java

```diff
@@ -56,8 +63,10 @@ public void init() throws IdGenerationException {
 
   @Override
   public void schemaRegistered(SchemaKey schemaKey, SchemaValue schemaValue) {
-    if (maxIdInKafkaStore.get() < schemaValue.getId()) {
-      maxIdInKafkaStore.set(schemaValue.getId());
-    }
+    String context = QualifiedSubject.contextFor(schemaRegistry.tenant(), schemaKey.getSubject());
```

Comment:
Similar to previous one, will schemaKey contain a tenant prefix? If not, the tenant will fall back to default?

Reply:
The subject will only contain a tenant prefix if the tenant is not the default.